[tiledscenesource, sensorthingssource, vectortilesource] select a new connection by default

ptitjano opened the pull request:
## Description

When a new connection is created, in most cases, this connection is gonna be loaded by the user. Therefore, it seems logic to select a new connection once it is created.

ptitjano committed:
qgstiledscenesourceselect: Select a new connection by default

When a new connection is created, in most cases, this connection is
gonna be loaded by the user. Therefore, it seems logic to select a new
connection once it is created.

nyalldawson commented on src/gui/tiledscene/qgstiledscenesourceselect.cpp:
```diff
@@ -93,6 +93,8 @@ void QgsTiledSceneSourceSelect::btnNewCesium3DTiles_clicked()
 
     QgsTiledSceneProviderConnection::addConnection( nc.connectionName(), connectionData );
     populateConnectionList();
+    QgsTiledSceneProviderConnection::setSelectedConnection( nc.connectionName() );
```
QgsSensorThingsSourceSelect and QgsVectorTileSourceSelect likely need a similar fix

ptitjano replied:
Indeed. This is now done.

ptitjano committed:
qgsvectortilesourceselect: Select a new connection by default

When a new connection is created, in most cases, this connection is
gonna be loaded by the user. Therefore, it seems logic to select a new
connection once it is created.

ptitjano committed:
qgssensorthingssourceselect: Select a new connection by default

When a new connection is created, in most cases, this connection is
gonna be loaded by the user. Therefore, it seems logic to select a new
connection once it is created.
